modules/storages/config/locales/en.yml

```diff
@@ -217,7 +217,9 @@ en:
         placeholder: Check your connection against the server.
         subtitle: Connection validation
         tenant_id_wrong: The configured directory (tenant) id is invalid. Please check the configuration.
-        unexpected_content: Unexpected content found in the drive.
+        unexpected_content:
+          nextcloud: Unexpected content found in the managed folder.
```

Comment:
🟡 not sure about the AC of this feature, but should we use the wording `in the managed group folder`? we do use it elsewhere, too.